src/components/vm/snapshots/vmSnapshotsCreateModal.tsx

```diff
@@ -127,11 +174,11 @@ const MemoryLocationRow = ({ onValueChanged, memoryLocation, validationError, av
     );
 };
 
-function get_available_space(path, superuser, callback) {
+function get_available_space(path: string, superuser: boolean, callback: (val: null | number) => void) {
     if (!path)
         callback(null);
 
-    cockpit.script(get_available_space_sh, [path], { superuser })
+    cockpit.script(get_available_space_sh, [path], superuser ? { superuser: "require" } : { })
```

Comment:
This added line is not executed by any test.

Reply:
FTR, `"superuser": false` is valid, but this is fine as well.